README.md

```diff
@@ -19,13 +19,37 @@ make build/cli
 A binary will be created in the root directory of the repo, which can be run:
 
 ```
-./cli
+./cluster-service
 ```
 
 
 ### How to use
+```bash
+# set env vars for clusters aws key and secret 
+export AWS_ACCESS_KEY_ID=<key value>
+export AWS_SECRET_ACCESS_KEY=<secret value>
+```
 
-TODO
+./cluster-service cleanup <cluster_id>
```

Comment:
It might be a tiny bit easier to read if this is in the code block as well

Reply:
:+1: